Neos.Neos/Configuration/Settings.yaml

```diff
@@ -481,7 +481,7 @@ Neos:
 
           noHomepageException:
             matchingExceptionClassNames:
-              - Neos\Neos\Routing\Exception\NoHomepageException
+              - Neos\Neos\FrontendRouting\SiteDetection\SiteDetectionFailedException
             options: *welcomeTemplate
```

Comment:
i have to evaluate if a `SiteDetectionFailedException` will with my adjustments even come that far ;)

Reply:
Yes the only case where this currently happens, is at the `neos/content` route, as the backend controller cannot find a site.

But in 8.3 we dont have a graceful error as well only something like `Site doest not exist for current user`: https://github.com/neos/neos-ui/issues/3182

We should probably redirect to some other available module, or at least keep the backend frame around and not destroy everything.

Reply:
i will remove the `noHomepageException` rendering group as this should be solve differently see https://github.com/neos/neos-ui/issues/3721